sentry_sdk/scope.py

```diff
@@ -441,13 +441,28 @@ def clear(self):
 
     @_attr_setter
     def level(self, value):
-        # type: (Optional[LogLevelStr]) -> None
-        """When set this overrides the level. Deprecated in favor of set_level."""
+        # type: (LogLevelStr) -> None
+        """
+        When set this overrides the level.
+
+        .. deprecated:: 1.0.0
+            Use :func:`set_level` instead.
+
+        :param value: The level to set.
+        """
+        logger.warning(
```

Comment:
@antonpirker Why are you emitting the warning with `logger`? When I have had to emit deprecation warnings, I usually do so as follows:
```python
warnings.warn("blah blah blah", DeprecationWarning, stacklevel=2)
```

Is there any significant difference with using `logger.warning` vs `warnings.warn`? Do we have a preference in the SDKs?

Reply:
Just used it because all the other deprecation warnings in that file do it the same way.